tests/plugins/dialog/positioning.js

```diff
@@ -0,0 +1,106 @@
+/* bender-tags: editor */
+/* bender-ckeditor-plugins: dialog, link, toolbar */
+
+( function() {
+	'use strict';
+
+	bender.editor = {};
+
+	bender.test( {
+		// When dialog is shown body has overflow hidden from class 'cke_dialog_open'
+		// and right padding equal to the scrollbars width. When there is predefined right padding
+		// it should be preserved. All extra styling is removed on dialog hide (#2395).
+		'test body has hidden scrollbars when dialog is opened': function() {
```

Comment:
Passes for me 😕.

Reply:
Ok I got it. Tests might fail with small browser window or landscape mode on mobiles. Thats because dialog can be moved only within viewport. I will fix it.